### PR TITLE
Updating README files

### PR DESCRIPTION
**Pull request by rorticus**
Updating README files. Adding missing information and sections for consistency with other packages.

### CODE REVIEW

File: todo-mvc-kitchensink/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+## How Do I Contribute?
+
+We appreciate your interest!  Please see the [Dojo Meta Repository](https://github.com/dojo/meta#readme) for the
+Contributing Guidelines and Style Guide.
+
+### Installation
+
+To start working with this package, clone the repository and run `npm install`.
+
+In order to build the project run `grunt dev` or `grunt dist`.
```
Comment (agubler) [inline todo-mvc-kitchensink/README.md]: we should remove this.

File: todo-mvc/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+## How Do I Contribute?
+
+We appreciate your interest!  Please see the [Dojo Meta Repository](https://github.com/dojo/meta#readme) for the
+Contributing Guidelines and Style Guide.
+
+### Installation
+
+To start working with this package, clone the repository and run `npm install`.
+
+In order to build the project run `grunt dev` or `grunt dist`.
```
Comment (agubler) [inline todo-mvc/README.md]: we should remove this (todo-mvc) doesn't have a `Gruntfile.js` anymore

Comment (rorticus) [inline todo-mvc/README.md]: Changed to `dojo build`

File: todo-mvc-kitchensink/README.md (hunk 1 above)
Comment (rorticus) [inline todo-mvc-kitchensink/README.md]: Changed to `dojo build`

File: README.md [hunk 3]
```diff
@@ -61,3 +58,41 @@ This assumes that npm & typings & dojo-cli have been installed for the example a
...
 
 The latest version(s) will now be available [on github pages](https://dojo.github.io/examples).
+
+## Features
+
+There are currently three examples:
+
+- [todo-mvc](./todo-mvc) - A reference implementation of [TodoMVC](http://todomvc.com/) built in current pre-release versions of Dojo 2 packages.
+- [todo-mvc-kitchensink](./todo-mvc-kitchensink) - A feature-rich version of TodoMVC, built in current pre-release versions of Dojo 2 packages.
+- [dojo-cli-example](./dojo-cli-example) - An example of working with Dojo cli
```
Comment (dylans) [inline README.md]: dojo-cli or @dojo/cli

File: README.md [hunk 4]
```diff
@@ -61,3 +58,41 @@ This assumes that npm & typings & dojo-cli have been installed for the example a
 ```
 
 The latest version(s) will now be available [on github pages](https://dojo.github.io/examples).
+
+## Features
+
+There are currently three examples:
+
+- [todo-mvc](./todo-mvc) - A reference implementation of [TodoMVC](http://todomvc.com/) built in current pre-release versions of Dojo 2 packages.
```
Comment (dylans) [inline README.md]: with instead of in?

File: dojo-cli-example/README.md [hunk 5]
```diff
@@ -1,27 +1,50 @@
 # dojo-cli-example
 
-<!-- TODO: change and uncomment
-[![Build Status](https://travis-ci.org/dojo/<< package-name >>.svg?branch=master)](https://travis-ci.org/dojo/<< package-name >>)
-[![codecov](https://codecov.io/gh/dojo/<< package-name >>/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/<< package-name >>)
-[![npm version](https://badge.fury.io/js/dojo-<< package-name >>.svg)](http://badge.fury.io/js/dojo-<< package-name >>)
--->
+An example of working with Dojo cli.
```
Comment (dylans) [inline dojo-cli-example/README.md]: dojo-cli or @dojo/cli

File: todo-mvc-kitchensink/README.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
+# Dojo2 TodoMVC Kitchen Sink Example
+
+A feature-rich version of TodoMVC, built in current pre-release versions of Dojo 2 packages.
```
Comment (dylans) [inline todo-mvc-kitchensink/README.md]: with instead of in ?

File: todo-mvc-kitchensink/README.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+- [Usage](#usage)
+- [Features](#features)
+- [How do I contribute?](#how-do-i-contribute)
+  - [Installation](#installation)
+  - [Testing](#testing)
+- [Licensing information](#licensing-information)
+
+## Pre-requisites
+
+You need to be running node `v6+` and npm `v3+`.
```
Comment (dylans) [inline todo-mvc-kitchensink/README.md]: v6 and v3 may confuse people, let's be more explicit please. And I thought we supported Node 4?

File: todo-mvc-kitchensink/README.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+To watch and serve the application source, then run the build with the `-w` command.
+
+```
+dojo build -w
+```
+
+## Features
+
+This package demonstrates how to build an application using Dojo2.
```
Comment (dylans) [inline todo-mvc-kitchensink/README.md]: Dojo 2

File: todo-mvc/README.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+- [Usage](#usage)
+- [Features](#features)
+- [How do I contribute?](#how-do-i-contribute)
+  - [Installation](#installation)
+  - [Testing](#testing)
+- [Licensing information](#licensing-information)
+
+## Pre-requisites
+
+You need to be running node `v6+` and npm `v3+`.
```
Comment (dylans) [inline todo-mvc/README.md]: Same as above.

File: todo-mvc/README.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+To watch and serve the application source, then run the build with the `-w` command.
+
+```
+dojo build -w
+```
+
+## Features
+
+This package demonstrates how to build a simple application using Dojo2 widgets.
```
Comment (dylans) [inline todo-mvc/README.md]: Dojo 2

File: README.md [hunk 11]
```diff
@@ -61,3 +58,41 @@ This assumes that npm & typings & dojo-cli have been installed for the example a
...
+- [todo-mvc-kitchensink](./todo-mvc-kitchensink) - A feature-rich version of TodoMVC, built in current pre-release versions of Dojo 2 packages.
+- [dojo-cli-example](./dojo-cli-example) - An example of working with Dojo cli
+
+Application examples that are deployed to [gh-pages](https://dojo.github.io/examples):
+
+ - [todoMVC](https://dojo.github.io/examples/todo-mvc)
+ - [todoMVC Kitchen Sink](https://dojo.github.io/examples/todo-mvc-kitchensink)
+
+## How Do I Contribute?
```
Comment (dylans) [inline README.md]: How do I contribute for consistency

File: dojo-cli-example/README.md [hunk 12]
```diff
@@ -1,27 +1,50 @@
...
-## How do I use this package?
+This project serves as an example on creating a custom command to use with `@dojo/cli`. The example shows:
 
-TODO: Add appropriate usage and instruction guidelines
+* Registering your command with `@dojo/cli`
+* Accepting arguments
+* Performing a simple task when the command is run
 
-## How do I contribute?
+## How Do I Contribute?
```
Comment (dylans) [inline dojo-cli-example/README.md]: Probably back to lowercase to keep consistency with the index?

File: todo-mvc-kitchensink/README.md [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+
+* Custom widgets using `@dojo/widget-core`
+* Applying CSS to widgets
+* Overriding CSS with theming.
+* Using `@dojo/router`
+* Using `@dojo/i18n` to provide internalization.
+
+## How Do I Contribute?
```
Comment (dylans) [inline todo-mvc-kitchensink/README.md]: Same as above

File: todo-mvc/README.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+
+It is an example of:
+
+* Custom widgets using `@dojo/widget-core`
+* Applying CSS to widgets 
+* Using `@dojo/router`
+
+## How Do I Contribute?
```
Comment (dylans) [inline todo-mvc/README.md]: same

**Comment by rorticus** [comment]
@dylans feedback has been addressed. I ended up deleting the bit on the node version because I think any reasonable version of node is going to work... we don't call it out for the other packages.

**Pull request merged**